### ISSUE TITLE
nanopub.org not working

### ISSUE THREAD

**Issue by nicola883** [Nanopublication/Guidelines#24]
Hi.
It seems that the website nanopub.org is not reachable:
http://nanopub.org/
I think that it is an essential resource for nanopublications.
Is it still being maintained?
Is there someone that can fix it?
Thank you!

**Comment by tkuhn** [comment]
Yes, it's a known issue. I didn't manage to figure out who exactly owns that domain, and those who I think might be the owners didn't respond. I will try again...

In the meantime, we have now this alternative website: https://nanopub.net
